Integrations/integration-HybridAnalysis.yml

```diff
@@ -354,230 +431,643 @@ script:
+            };
+            urlContext.Malicious = maliciousContext;
+        }
+
+        var context = {};
+
+        if (getIntegrationContext()[url] == 'file_url') {
+            context = createFileContext (url, response, dbotScore, baseContext, urlContext, DbotContext, maliciousContext);
+        }
+
+        else {
+            DbotContext.Indicator = url;
+            context = {
+                'HybridAnalysis.URL(val.ScanID && val.ScanID == obj.ScanID)' : baseContext,
+                'URL(val.Data && val.Data == obj.Data)' : urlContext,
+                'DBotScore' : DbotContext
+            }
+        }
+
+        return context;
+    }
+
+
+    function createQuickScanHumanReadable (response, scannersContext, whitelist, headers){
+        var mdTable = {
+            'ScanID': response.id,
+            'SHA256': response.sha256,
+            'Finished': response.finished
+        };
+        var md = '### Scan Results:\n' + tableToMarkdown(null, mdTable, ['ScanID', 'SHA256', 'Finished'], undefined, headerTransform=undefined, removeNull=true);
+        var scanTable = '##### scanners:\n' + tableToMarkdown(null, scannersContext, headers, undefined, headerTransform=undefined, removeNull=true);
+        var whitelistTable = '##### whitelist:\n' + tableToMarkdown(null, whitelist, headers, undefined, headerTransform=undefined, removeNull=true);
+
+        var humanReadable = md + scanTable + '\n' + whitelistTable;
+        return humanReadable;
+    }
+
+
+        // create scanners entry context and calculate counters
+    function createScannersContext (response) {
+        var unknownCounter = 0;
+        var maliciousCounter = 0;
+        var maliciousDescription = "";
+        var scannersContext = [];
+
+        response.scanners.forEach(function(key) {
+            scannersContext.push(
+                {
+                    'Name': key.name,
+                    'Status': key.status,
+                    'Positives': key.positives
+                });
+            if (key.status == 'unknown' || key.status =='no-result' || key.status == 'not-supported')
+                unknownCounter++;
+            else if (key.status == 'malicious'){
+                maliciousCounter++;
+                maliciousDescription = maliciousDescription + ', ' + key.name;
+            }
+        });
+
+        maliciousDescription = maliciousDescription.substr(2); // remove the first ", "
+        var res = {
+            'scannersContext' : scannersContext,
+            'unknownCounter' : unknownCounter,
+            'maliciousCounter' : maliciousCounter,
+            'maliciousDescription' : maliciousDescription
+        }
+
+        return res;
+    }
+
+
+    //create table entry from response after 'quick-scan-url-results' command
+    function quickScanResultToEntry(response, minMaliciousScanners) {
+
+        var scannersContext = createScannersContext (response)['scannersContext'];
```

Comment:
you are running the same function 4 times with the same param, better to run it once and extract the info later